### PR TITLE
Update docs to changed Python type

### PR DESCRIPTION
**Pull request by doosuu**
## Describe your changes

Update the documentation to mention the newly introduced `TypedDataPointResult` and its members `value` and `timestamp`

Corresponds with Python SDK version 0.7.0

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by dennismeister93 — APPROVED** [review]
Tested docu locally, looks good

**Review by emsameen — APPROVED** [review]
There are still few places where the DataPointReply is used and can be adapted as well.
- docs/content/en/docs/about/development_model/vehicle_app_sdk/_index.md

**The rest LGTM**

File: content/en/docs/about/repository-overview/_index.md [hunk 1]
```diff
@@ -12,14 +12,15 @@ description: >
...
+| [vehicle-app-python-template](https://github.com/eclipse-velocitas/vehicle-app-python-template)   | GitHub Template repository containing an exemplary Vehicle App that uses an exemplary SDK to provide access to vehicle data points and methods. The sample SDK extends the _sdv-vehicle-app-python-sdk_. In addition the template repository contains the development environment for Visual Studio Code for a Vehicle App as well as the CI/CD workflows that can be used as blueprint for your own Vehicle App written in Python.
+| [vehicle-app-python-sdk](https://github.com/eclipse-velocitas/vehicle-app-python-sdk)             | Provides basic functionality to write an SDK to allow access to vehicle data points and method. This includes publishing & subscribe messaging, talent abstraction, vehicle data model ontology and function-based query & rule support.
```
[the remark this answers is not in the document]
Comment (doosuu) [reply content/en/docs/about/repository-overview/_index.md:16]: Done

File: content/en/docs/about/repository-overview/_index.md [hunk 2]
```diff
@@ -12,14 +12,15 @@ description: >
...
 | [vehicle-model-python](https://github.com/eclipse-velocitas/vehicle-model-python)                 | Basic vehicle model for Python generated from [VSS](https://github.com/COVESA/vehicle_signal_specification/tree/master/spec) with addition of some specialized vehicle services.
-| [vehicle-app-cpp-sdk](https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk)                   | Provided basic functionality to write a Vehicle App which has access to vehicle data points and services. This includes publish & subscribe messaging, talent abstraction, vehicle data model ontology and function-based query & rule support.
```
[the remark this answers is not in the document]
Comment (doosuu) [reply content/en/docs/about/repository-overview/_index.md:18]: Done

**Pull request merged**